pandas/io/excel/_calaminereader.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
+
+from datetime import (
+    date,
+    datetime,
+    time,
+)
+from tempfile import NamedTemporaryFile
+from typing import Union
+
+from pandas._typing import (
+    FilePath,
+    ReadBuffer,
+    Scalar,
+    StorageOptions,
+)
+from pandas.compat._optional import import_optional_dependency
+
+import pandas as pd
+
+from pandas.io.common import stringify_path
+from pandas.io.excel._base import (
+    BaseExcelReader,
+    inspect_excel_format,
+)
+
+ValueT = Union[int, float, str, bool, time, date, datetime]
```

Comment:
Can we stick with the same `Scalar` import that the other readers use?

Reply:
This type is for internal using in `_convert_cell`. `get_sheet_data` return `Scalar`.

Reply:
Can you move this into the function it is actually used in? Sounds like it has a pretty localized use, so no need to be in the global namespace

Reply:
Renamed in _CellValueT.